app/provider/app.go

```diff
@@ -628,6 +635,8 @@ func New(
 		providertypes.ModuleName,
 	)
 
+	// NOTE: provider module needs to come after the staking module, since
+	// it needs the information the staking module provides to compute validator updates.
```

Comment:
Do we need to have the `no_valupdates_staking` module's `EndBlocker` here as well?

Reply:
No, this just sets the order of EndBlockers by stating the names of the modules; this is ok to say staking.
Thanks for double-checking